enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/JWTUtil.java

```diff
@@ -131,5 +154,98 @@ public static boolean verifyTokenSignature(SignedJWT jwt, String alias) throws M
         }
     }
 
+    public static PrivateKey getPrivateKey() {
+        PrivateKey privateKey = null;
+        try {
+            String strKeyPEM = "";
+            BufferedReader br = new BufferedReader(new FileReader(ConfigHolder.getInstance().getConfig().
+                    getPrivateKeyPath()));
+            String line;
+            while ((line = br.readLine()) != null) {
+                strKeyPEM += line + "\n";
+            }
+            br.close();
+            strKeyPEM = strKeyPEM.replace(Constants.BEGINING_OF_PRIVATE_KEY, "");
+            strKeyPEM = strKeyPEM.replaceAll(System.lineSeparator(), "");
+            strKeyPEM = strKeyPEM.replace(Constants.END_OF_PRIVATE_KEY, "");
```

Comment:
do we need to read line by line to the string? can't we readall line at once to the string and replace?
```suggestion
            Path keyPath = Paths.get(ConfigHolder.getInstance().getConfig().getPrivateKeyPath());
            String key = Files.readString(keyPath, Charset.defaultCharset());
            strKeyPEM = key
                .replace(Constants.BEGINING_OF_PRIVATE_KEY, "")
                .replaceAll(System.lineSeparator(), "")
                .replace(Constants.END_OF_PRIVATE_KEY, "");
```